src/pyvalues.c

```diff
@@ -541,9 +568,22 @@ static meta_data_t *cpy_build_meta(PyObject *meta) {
     Py_DECREF(key);
   }
   Py_XDECREF(l);
+  return 0;
+}
+
+CPY_BUILD_META_HANDLER(meta_data, meta_data_t);
+CPY_BUILD_META_HANDLER(plugin_notification_meta, notification_t);
+
+static meta_data_t *cpy_build_value_meta(PyObject *meta) {
```

Comment:
I think this pretty wrapper should get the pretty `cpy_build_meta()` name. The other function could be renamed to `cpy_build_meta_generic()` to underline that it works with more than one meta type.